examples/gno.land/p/demo/acl/gno.mod

```diff
@@ -0,0 +1,6 @@
+module gno.land/p/demo/acl
+
+require (
+        "gno.land/p/demo/avl" v0.0.0
```

Comment:
Can we use 'latest' instead of 'v0.0.0' to accurately reflect potential variations across computers?

Reply:
we are parsing CanonicalVersion(just like `go.mod`). 

Go replaces the `latest` with the latest version number before parsing go.mod file (using go proxy). We might need to support versioning before supporting keyword `latest`

Reply:
What’s the issue with having `latest` now and later implement versioning?

Reply:
Just for context, Let's start with how `latest` works in go.mod.

modfile's(golang.org/x/mod/modfile) parsing logic doesn't understand `latest`. It only understands CanonicalVersion. Since we are using the same logic for gno.mod we can't parse `latest` as version.

What `go` does is, as soon as you run a go command it replaces the `latest` with actual latest version in the original `go.mod` file using go-proxy and then parses the `go.mod` file.

IOW you can write `latest` in a require directive, but the next time you run a go command, it will replace the word `latest` with the actual version. It will not stay `latest`. 

--- 

So first we need a proxy that resolves `latest` to the actual version. (We can obviously have a dummy proxy that always resolves to `v0.0.0`, but I won't recommend it)

Plus it's not a good idea to modify gno.mod file automatically before someone runs `gno mod download`. you wouldn't have deterministic dependencies this way.

Reply:
Imagine someone running `gno test ./example` command and it modifies all the gno.mod file 💀

Reply:
> What go does is, as soon as you run a go command it replaces the latest with actual latest version in the original go.mod file using go-proxy and then parses the go.mod file.

What about making this later, and keeping `latest` for now?

Reply:
Here: df872c08931c45bd8566aeaff3a5c991a981905e